resume.md

```diff
@@ -1,49 +1,47 @@
-# FIRST LAST NAME
+# Kelly Ma
 
-(123) 456 - 7890 | email@gmail.com
+(510) 599 - 0540 | kellyma@berkeley.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+https://kelly2135ma.wixsite.com/mysite | https://www.linkedin.com/in/kelly-ma-2a1172174/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**Galileo Academy** | *June 2020*
+**University of College** | B.S. Degree ... *May 2024*
+* **GPA**:4.0 | 
+* **Coursework**: Introduction to Programming Data Science Foundations, Linear Algebra.
```

Comment:
Use the generalized names - when you do more upper div classes this section should only be those.

* ~~Introduction to Programming Data Science Foundations~~ --> Data Science Foundations
* Linear Algebra
* Multivariable Calculus (Planned) blah blah

Reply:
Consider putting Data Structures + Algorithms because that is the most important class on a technical resume.